packages/apollo-datasource-rest/src/RESTDataSource.ts

```diff
@@ -218,11 +218,12 @@ export abstract class RESTDataSource<TContext = any> extends DataSource {
       url.searchParams.append(name, value);
     }
 
-    // We accept arbitrary objects as body and serialize them as JSON
+    // We accept arbitrary objects and arrays as body and serialize them as JSON
     if (
       options.body !== undefined &&
       options.body !== null &&
       (options.body.constructor === Object ||
+        options.body.constructor === Array ||
```

Comment:
It's obviously not common to sub-class an `Array`, but could/should we switch this to use [`Array.isArray`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray) instead to help accommodate that?

Reply:
@abernix Works for me. Updated.